lib/_http_client.js

```diff
@@ -632,20 +642,19 @@ function responseOnEnd() {
         socket.end();
     }
     assert(!socket.writable);
-  } else if (req.finished) {
-    // We can assume `req.finished` means all data has been written since:
+  } else if (req.writableEnded) {
```

Comment:
Ideally, the property is also documented in the regular documentation instead of only being mentioned in the deprecation notice.

Reply:
@BridgeAR It is documented under `Writable`?